packages/autocomplete-core/src/createAutocomplete.ts

```diff
@@ -58,6 +58,9 @@ export function createAutocomplete<
       setIsOpen,
       setStatus,
       setContext,
+      nextState: {
+        isOpen: store.getState().isOpen,
```

Comment:
is there other values of state that should be kept?

Reply:
Haven't found any yet.
I'll send a new PR when I find something.